### ISSUE TITLE
Issues with new design changes

### ISSUE THREAD

**Issue by rabiibrahimi** [mautic/mautic#14321]
### Mautic Series

5.1.x series

### Mautic installed version

5.2

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.2

### What browsers are you seeing the problem on?

Firefox, Chrome

### What happened?

Just the new design not properly sorted. issue with how things renders. e.g installaton page the forms is it not centered and fields not displayed properly like dropdown. attached screenshots 

<img width="949" alt="install-one" src="https://github.com/user-attachments/assets/f3715eb8-e55b-4780-ae14-5259542cb1ba">
<img width="449" alt="lack-of-consistency" src="https://github.com/user-attachments/assets/ee524667-e158-4d01-851e-2a007e36197d">
<img width="960" alt="install-db" src="https://github.com/user-attachments/assets/6a244425-c047-4b8a-8255-004edea36fdf">

### How can we reproduce this issue?

Step 1: Download the version 5.2 and install it and check the issues with design.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by rabiibrahimi** [comment]
tbh the new style applied to mautic is not good. There is nothing better than clear simple UI with few clean colors. Is there is anyway to keep using old styles in newer version ?

**Comment by RCheesley** [comment]
Thanks @rabiibrahimi I am sure this is a quick fix, honestly I think we probably just completely overlooked the installation screens as mostly our testing and UX / UI reports are based on a Mautic instance which is already installed.

cc @mautic/ux-ui-tiger-team

**Comment by mohamed-badaoui** [comment]
I just test migration from 5.1.1 to 5.2, and ui is broken
the ui is broken for all the pages with a list like contact, email, campaign ... 

see https://github.com/mautic/mautic/issues/14324

![image](https://github.com/user-attachments/assets/8c6c65b9-c666-4906-bb56-76c292854d11)

**Comment by dsp76** [comment]
@mohamed-badaoui can't confirm a general layout problem. try to do a console mautic:assets:generate and a cache clear

**Comment by mohamed-badaoui** [comment]
> @mohamed-badaoui can't confirm a general layout problem. try to do a console mautic:assets:generate and a cache clear

I did but nothing changes:
php bin/console mautic:assets:cleanup
php bin/console mautic:assets:generate
php bin/console cache:clear

**Comment by rabiibrahimi** [comment]
Also on contact page there are some issue that need to be fixed. see attached screenshot
There is an issue with spacing there, some padding needs to be added for each stats box so that the text and the percentage are not stuck to the right. Also the border bottom of the title is stuck to the div below (IMO) it doesn't serve anything and should be removed. Issue with border on right near contact's avatar and finally the message below the stats should have some margin applied. 

<img width="960" alt="mautic-issue-design-contact" src="https://github.com/user-attachments/assets/afdf7c3d-63e4-433b-8cd6-73cbc78895e7">

**Comment by andersonjeccel** [comment]
@mohamed-badaoui if you’re using Windows, hit Ctrl + Shift + R on the Mautic page or clear manually your browser’s cache

@RCheesley clearing the browser's cache is something everyone has to do with their own computer when migrating to the new version, it might be good to make an announcement

**Comment by andersonjeccel** [comment]
@mohamed-badaoui something is unusual on your specific installation

try to:
0. Open using incognito browsing (this will tell if the issue relies on the browser or server)
1. Manually open the server storage and delete the var/cache folder
2. Delete the CSS files located in media/(something)/
3. Run the command to generate assets only after step 2 and check if the deleted files appear again
4. Check your env.local file to ensure it’s prod env
5. Try switching env to dev temporarily to see if the issue is gone
6. Delete all folders (except the one responsible for uploaded stuff, probably /media) and reinstall

**Comment by stefan-franz** [comment]
I have after an update always to set the rights - i had also strange UI. After setting the rights, it works: 
(use your own mautic path instead of mine)
Maybe it helps you
chown -R www-data:www-data /var/www/html/mautic
chmod -R 755 /var/www/html/mautic

**Comment by andre-mha** [comment]
Just done a complete fresh installation and going through each page to spot issues, I also have the same issues as initially reported and have found that the builder on landing pages defaults to the legacy builder instead of grapgejs and no editing can be done.

![Annotation 2024-12-05 101711](https://github.com/user-attachments/assets/d5cd3b60-dce1-4044-ba9b-19b8fbbbec2c)

**Comment by andre-mha** [comment]
The "choose" field on uploading part on themes overlaps with the install button 

![image](https://github.com/user-attachments/assets/6573cda8-3bae-4f91-8fb2-bed32e008ee6)

**Comment by mohamed-badaoui** [comment]
@andersonjeccel @stefan-franz 
Thank you very much for your help, Problem solved.

**Comment by rabiibrahimi** [comment]
here is another issue with the design.

<img width="960" alt="mautic-builder" src="https://github.com/user-attachments/assets/256db516-8c95-49db-843f-1abca6a1902f">
<img width="472" alt="no-space-click-event" src="https://github.com/user-attachments/assets/efade6c9-aa3c-420b-a011-6339377171d4">
